packages/nuxt-module/plugins/i18n-cookies.js

```diff
@@ -47,7 +47,7 @@ const i18nCookiesPlugin = ({ $cookies, i18n, app, redirect }) => {
   if (isServer) {
     app.i18n.cookieValues = {
       ...(autoChangeCookie.locale && { [cookieNames.locale]: targetLocale }),
-      ...(autoChangeCookie.currency && { [cookieNames.currency]: getCurrencyByLocale(targetLocale) })
+      ...(autoChangeCookie.currency && { [cookieNames.currency]: app.$config.defaultCurrency || getCurrencyByLocale(targetLocale) })
```

Comment:
You can wrap this in `getDefaultCurrency` functions. It might seem like something that is not required but it will leave us with a more declarative code and there will be one place where default currency is resolved.